peps/pep-0740.rst

```diff
@@ -49,6 +49,40 @@ expressed by both package maintainers and downstream users:
 
 This proposal seeks to accommodate each of the above use cases.
 
+Additionally, this proposal identifies the following motivations:
+
+* Verifiable provenance for Python package distributions: many Python
+  packages currently contain *unauthenticated* provenance metadata, such
+  as URLs for source hosts. A cryptographic attestation format could enable
+  strong *authenticated* links between these packages and their source hosts,
+  allowing both the index and downstream users to verify that a package
+  truly originates from its claimant repository.
```

Comment:
super-nit: TIL that "claimant" is a word, perhaps we should stick with simpler terminology where possible?

Reply:
Sounds good; how does "claimed" sound? Alternatively, I could reword this as "allowing both the index and downstream users to cryptographically verify that a package originates from a specific source repository" (or similar).

Reply:
Claimed sounds good to me!